build_scripts/npm_macos/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "appdmg": "^0.6.4",
     "electron-installer-dmg": "^3.0.0",
-    "electron-osx-sign": "^0.5.0",
+    "electron-osx-sign": "^0.6.0",
```

Comment:
This causes NPM CI to fail, because its not also updating the lock file - Lock file needs to be updated as well before this can be merged

https://github.com/Chia-Network/chia-blockchain/runs/7873327082?check_suite_focus=true#step:19:31

Reply:
Updated the lock file